fastlane/lib/fastlane/actions/delete_keychain.rb

```diff
@@ -4,25 +4,17 @@ module Fastlane
   module Actions
     class DeleteKeychainAction < Action
       def self.run(params)
-        original = Actions.lane_context[Actions::SharedValues::ORIGINAL_DEFAULT_KEYCHAIN]
-
-        search_paths = []
-        search_paths << File.expand_path(params[:name]) if params[:name]
-        search_paths << File.expand_path(File.join("~", "Library", "Keychains", params[:name])) if params[:name]
-        search_paths << File.expand_path(params[:keychain_path]) if params[:keychain_path]
```

Comment:
We could, but the realization I made is that this is what the `Helper` method does. It felt better to keep this name discovery logic consistent in that one place.

Does that make sense, or am I misunderstanding?